libraries/browser-client/src/websocket/worker.ts

```diff
@@ -321,9 +324,8 @@ class WebsocketClient {
   }
 
   send(data: string | Int16Array): void {
-    if (!this.websocket) {
-      throw new Error('No Websocket')
-    }
+    // We're offline, most likely due to an earlier error
+    if (!this.websocket) return
```

Comment:
Is this now a place where a problem might be hidden? So if the `websocket` is falsy, nobody will know? Would it make sense to throw an Error from here, which might make the whole stack to collapse and that would expose the potential problem?

Reply:
The websocket close has been detected earlier by WebsocketClient.onWebsocketClose and logged as an error. Removing the throw is to suppress redundant error messages. There may be audio packets still in the postMessage system that may trigger send. Setting websocket to falsy is used to signal that the websocket is no longer available. I think throw can be restored here at the cost of an error spree.

Reply:
I'll rerun some tests with the throw.

Reply:
Restored the throw.